README.md

```diff
@@ -1 +1,48 @@
-# TodoList
\ No newline at end of file
+# TodoList
+This is a project about to-do's . Projects can be added and tasks(to-do's) added under each project.  The priorities chosen will display a different background color for any todo(task)  The to-do's and projects will be stored on local storage.
+ 
+![screenshot](./dist/assets/images/todoshotOne.png)
```

Comment:
## Improve the screenshot.
- As part of having and maintaining a professional repo, the README, being the gateway to the project ought to stand out at all time as it is what sells the project the most. 
- [ ] Please ensure that the screenshot is showing here as expected.